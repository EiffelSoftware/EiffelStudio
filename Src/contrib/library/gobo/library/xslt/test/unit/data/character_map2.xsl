<?xml version="1.0" encoding="ISO-8859-1" ?>
<xsl:transform 
  xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
  version="2.0">

<xsl:import href="./character_map3.xsl" />
<xsl:output method="html" escape-uri-attributes="yes" use-character-maps="html-doc" />

</xsl:transform>
