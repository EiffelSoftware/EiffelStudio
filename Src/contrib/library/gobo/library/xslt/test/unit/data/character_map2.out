<html>
   <body><img src="./a-url-that-needs-escaping-because-of-%EE%90%85-is-in-the-path-although-the-following-should-remain-as-an-entity-&amp;nbsp;"><p>
   &nbsp;fred&iexcl;jim&t-and-c;harry&chap1;jill&chap2;tab-to-three-spaces   CRLF

      </p>
   </body>
</html>